fastchat/serve/controller.py

```diff
@@ -224,6 +224,56 @@ def worker_api_generate_stream(self, params):
             }
             yield json.dumps(ret).encode() + b"\0"
 
+    def worker_api_generate_completion(self, params):
+        worker_addr = self.get_worker_address(params["model"])
```

Comment:
seems a lot of duplicate code, try to create some common utility functions for them.

Reply:
duplicated codes are packed up now